components/definitions/rewardsExtensions.d.ts

```diff
@@ -1,7 +1,7 @@
 declare namespace RewardsExtension {
   interface State {
     balance: Balance
-    currentNotification?: string
+    currentNotification?: string | number
```

Comment:
@zenparsing 
 `components/brave_rewards/resources/extension/brave_rewards/background/reducers/rewards_panel_reducer.ts`

currentNotification is being assigned to a number in which ts complains
> Type ‘number’ is not assignable to type ‘string | undefined’.

To fix the above issue I modified the type definition itself rather than the implementation. If you feel this needs to be addressed more, please feel to do so.